c7n/resources/vpc.py

```diff
@@ -1856,6 +1856,23 @@ class resource_type(query.TypeInfo):
         id_prefix = "igw-"
 
 
+@InternetGateway.action_registry.register('delete')
+class DeleteInternetGateway(BaseAction):
+
```

Comment:
Could you add a doc string here? That will help with schema output and with example policies on the doc site.